Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -114,4 +142,11 @@ select.remotion-themed::-webkit-scrollbar-thumb,
 {
   background-color: var(--color-primary);
   border-radius: var(--remotion-themed-dropdown-scrollbar-border-radius);
+}
+
+.remotion-themed.readOnly .content span
```

Comment:
This one is too specific. `.content span` is a very specific implementation thingy